chia/timelord/timelord.py

```diff
@@ -91,6 +91,7 @@ def __init__(self, root_path, config: Dict, constants: ConsensusConstants):
         self.sanitizer_mode = self.config["sanitizer_mode"]
         self.pending_bluebox_info: List[Tuple[float, timelord_protocol.RequestCompactProofOfTime]] = []
         self.last_active_time = time.time()
+        self.process_id: int = 0
```

Comment:
Is this misnamed - because I was initially thinking this was an actual process id. But it's just an internal counter?